pages/docs/some-react-code/index.jsx

```diff
@@ -3,18 +3,18 @@ import Demo from './_Demo'
 import DocumentTitle from 'react-document-title'
 import { config } from 'config'
 
+exports.data = {
+  title: 'Some React Code',
+  date: `${new Date()}`
```

Comment:
Yes — that's a confusing part of this — it's in a JS file but it's static. We could potentially eval the data file but that'd only partially help as it'd add additional sources of confusion e.g. referencing other variables wouldn't work.